test/NuGet.Core.Tests/NuGet.CommandLine.Xplat.Tests/MSBuildAPIUtilityTests.cs

```diff
@@ -174,14 +177,15 @@ public void AddPackageReferenceIntoProjectFileWhenItemGroupDoesExist_Success()
             project.Save();
 
             // Assert
-            Assert.Contains(@$"<PackageReference Include=""X"" />", File.ReadAllText(Path.Combine(testDirectory, "projectA.csproj")));
-            Assert.DoesNotContain(@$"<Version = ""1.0.0"" />", File.ReadAllText(Path.Combine(testDirectory, "projectA.csproj")));
+            string updatedProjectFile = File.ReadAllText(Path.Combine(testDirectory, "projectA.csproj"));
+            Assert.Contains(@$"<PackageReference Include=""X"" />", updatedProjectFile);
+            Assert.DoesNotContain(@$"<Version = ""1.0.0"" />", updatedProjectFile);
```

Comment:
This always return true, so I think it should be changed to $"PackageReference Includes =""X"" <Version = ""1.0.0"" />". This is along the lines of this comment: https://github.com/NuGet/NuGet.Client/pull/4700#discussion_r915421010